project/home/templates/home/home_page.html

```diff
@@ -9,5 +9,12 @@
 <p>Welcome to Wagtail Social.</p>
 
 <div class="intro">{{ page.intro }}</div>
+{% for invite_friend in invite_friends %}
+    {% with invite_friend=invite_friend.specific %}
+        <div style="text-align: right;">
```

Comment:
Bootstrap has [utility classes to float elements](https://getbootstrap.com/docs/4.0/utilities/float/#classes).

```suggestion
        <div class="float-right">
```